### PR TITLE
QPPSF-8019: Y5 eCQM Data Update

### PR DESCRIPTION
**Pull request by NateTheGreatt**
#### Motivation for change

Added eCQM zip for performance year 2021 (`staging/2021/EP-EC-eCQM-2020-05.zip`, filename for 2021 is a year prior for some reason) and ingested the data.

Had to manually fix data for measure `CMS156v9` due to it becoming a `singlePerformanceRate` measure in Y5:
https://ecqi.healthit.gov/ecqm/ep/2021/cms156v9

#### What is being changed

Detailed information about what is being changed and rationale for interesting decisions made

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [ ] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [ ] Documentation updated
* [ ] Unit tests added/passing
* [ ] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPA-8019

### CODE REVIEW

File: util/measures/2021/additional-stratifications.json [hunk 1]
```diff
@@ -1 +1,2 @@
-[]
```
Comment (saquino0827) [inline util/measures/2021/additional-stratifications.json:1]: I see some missing stratifier info: CMS74v10, CMS137v9, CMS146v9, CMS153v9, CMS154v9, CMS155v9, CMS159v9.
You'll need to match the measure's numerator UUID and matching strata UUIDs together similar to the previous year: https://github.com/CMSgov/qpp-measures-data/blob/develop/util/measures/2020/additional-stratifications.json

File: util/measures/2021/cpc+-measure-groups.json [hunk 2]
```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "Outcome_Measure": ["CMS122v8", "CMS165v8"]
```
Comment (saquino0827) [inline util/measures/2021/cpc+-measure-groups.json:2]: Do these need to be the v9 versions?

**Review by saquino0827 — CHANGES_REQUESTED** [review]
A couple of things to change but overall mostly done. Nice work1

File: util/measures/2021/additional-stratifications.json (hunk 1 above)
Comment (NateTheGreatt) [reply util/measures/2021/additional-stratifications.json:1]: whoops, i added this but accidentally removed it in one of the commits lol

File: util/measures/2021/cpc+-measure-groups.json (hunk 2 above)
Comment (NateTheGreatt) [reply util/measures/2021/cpc+-measure-groups.json]: good catch!

**Review by saquino0827 — APPROVED** [review]
Lgtm, nice job!

**Review by KyleApfel — APPROVED** [review]

**Pull request merged**